frontend-v2/src/features/model/VariableRow.tsx

```diff
@@ -253,6 +253,7 @@ const VariableRow: FC<Props> = ({
   const noMapToPD = isPD || effectVariable === undefined || !isConcentration;
   const noDerivedVariables = !isConcentration || isPD;
   const isC1 = model.is_library_model && variable.qname.endsWith(".C1");
+  const disableAuc = false;
```

Comment:
I added this for consistency with the other derived variables, but I'm not sure if there's a scenario where we would want to disable secondary parameters for a concentration variable.